ufo/handlers/routes.py

```diff
@@ -15,8 +15,12 @@
 def landing():
   return flask.render_template('landing.html')
 
-@ufo.app.route('/vulcanized')
-def vulcanized():
+@ufo.app.route('/vulcanized_html')
+def vulcanized_html():
```

Comment:
nit: Can we add a comment here to document that the no caching is to avoid i18n problem?